pandas/core/indexes/multi.py

```diff
@@ -1035,11 +1035,13 @@ def is_lexsorted(self):
         """
         return self.lexsort_depth == self.nlevels
 
-    def is_lexsorted_for_tuple(self, tup):
+    def _true_slices_indices(self, tup):
```

Comment:
why wouldn't this just return True if a slice, False if not? IOW a list of booleans.